myfafco/settings.py

```diff
@@ -5,7 +5,7 @@
 from dotenv import load_dotenv
 
 load_dotenv()
-
+BASE_URL = 'http://127.0.0.1:8000/'

```

Comment:
again added the hardcoded data

Reply:
kyuki maine password reset k liye email send kraya h and without it shows error , if it has another way i will try once again

Reply:
Actually, you have provided the hardcoded data, but you need to add the data inside the environment file and get it from there.